### PR TITLE
Global workflow env

### PR DESCRIPTION
**Pull request by kgerheiser**
Add global-workflow-env to test specs.

This most specifically adds MET/METplus.

I have tested this pretty extensively on a fresh Ubuntu 20.04 VM with GCC/Intel, and my Mac 12.4 with GCC/Clang. The Ubuntu GCC runner finished, and the other ones are still going.

https://github.com/kgerheiser/spack-stack/actions

### CODE REVIEW

**Review by climbfuji — APPROVED** [review]

**Comment by climbfuji** [comment]
Let's wait for the CI tests to finish before merging.

**Comment by kgerheiser** [comment]
All workflows successfully completed.

**Pull request merged**
